src/persist/watcherUpdatable.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 The Casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Watcher } from './watcher';
+
+export interface watcherUpdatable extends Watcher {
```

Comment:
Why add watcherUpdatable ? I didn't see it's used anywhere?

Reply:
The Go version add this, so I add this interface